fsharp-backend/src/LibBackend/Account.fs

```diff
@@ -419,5 +454,6 @@ let initializeDevelopmentAccounts (serviceName : string) : Task<unit> =
     print $"Initing LibBackend.Account in {serviceName}"
     do! initTestAccounts ()
     do! initAdmins ()
+    print $"Inited  LibBackend.Account in {serviceName}"
```

Comment:
Is `print $"Initing LibBackend.Account in {serviceName}"` here twice intentionally?

Reply:
nevermind. I read too quickly.

Reply:
One says "inited" and the other says "initing"